File: src/main/java/org/fletchly/genius/ollama/model/OllamaResponse.java
package org.fletchly.genius.ollama.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Builder;

@Builder
@JsonIgnoreProperties(ignoreUnknown = true)
public record OllamaResponse(
        String model,
        String createdAt,
        OllamaMessage message
) {
}

File: src/main/kotlin/io/fletchly/genius/ollama/model/OllamaResponse.kt
package io.fletchly.genius.ollama.model

import io.fletchly.genius.conversation.model.Message
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class OllamaResponse(
    val model: String,
    @SerialName("created_at")
    val createdAt: String,
    val message: Message
)
